__tests__/app.test.js

```diff
@@ -36,3 +36,41 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/reviews/review_id", () => {
+  test("GET: 200 - returns object with the relevant ID", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review).toMatchObject({
```

Comment:
We can check against the specific review contents here rather than just expect.any.

Reply:
So is it better to input specific data from the test database for each key. My thinking was that only the review_id needs to be checked in this test and so long as the object contains the other keys the info inside it didn't matter?

Or are there other expect.somethings that would work better here?